src/vim.rs

```diff
@@ -275,4 +281,11 @@ impl Vim {
     pub fn set_signs(&self, filename: &str, signs: &[Sign]) -> Result<i8> {
         self.rpcclient.call("s:set_signs", json!([filename, signs]))
     }
+
+    pub fn update_tagstack(&self, winnr: Winnr, item: TagStackItem) -> Result<()> {
```

Comment:
I'm pretty sure `settagstack` is a `neovim` only function, so I think it would be a good idea to make this return early if it's running `vim`.

Reply:
This works in recent versions of vim8, including the one I am running.

Reply:
ah fair enough! :+1:

Reply:
We may still need to add checks for users of older versions. I'm not sure how to do that yet tho.

Reply:
You can use `exists('*settagstack')` for that. Might be worth creating a `SetTagStack` function to do all the getting the current stack, clearing it, and setting the new one plus checking if it's supported all in one function and one call from rust to vim too.

Reply:
That's what this function does already, so I'd only need to add those `exists()` calls. I can figure out how to do that.

Reply:
What I mean is, the function you've set up now issues two rpc messages to vim, one for `gettagstack` and another one for `settagstack` after clearing and setting the new stack. Adding the `exists` call will be a third rpc message sent to vim, so I'm wondering if it might be worth creating a vimscript function that does this instead and calling that instead, so that there's a single rpc message being sent to vim, and there's less network overhead for something that should be relatively simple otherwise. Maybe it's not worth the code complexity though, but worth keeping in mind if it ends up doing more things that require sending extra messages.